openpype/settings/defaults/project_settings/photoshop.json

```diff
@@ -44,6 +44,22 @@
             }
         }
     },
+    "scriptsmenu": {
+        "name": "OpenPype Tools",
+        "definition": [
+            {
+                "type": "action",
+                "command": "import sys; sys.stdout.write('Hello');",
```

Comment:
Would this actually work? What kind of scripts are you expecting here? JSX that modifies .psd? Could you please elaborate?